viewer/src/main/webapp/viewer-html/components/Search.js

```diff
@@ -56,6 +56,7 @@ Ext.define ("viewer.components.Search",{
     showSearchButtons: true,
     simpleSearchResults: false,
     searchFieldTriggers: null,
+    pickerAlignment: "tl-bl?", // Ext default
```

Comment:
does this `?` belong here?

Reply:
seems it does.. https://docs.sencha.com/extjs/6.2.0/classic/Ext.util.Positionable.html#method-alignTo